### PR TITLE
Add organization page

### PR DESCRIPTION
**Pull request by frinyvonnick**
Related #164

### CODE REVIEW

File: src/screens/organizer/organization/components/organizationForm/organizationForm.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { Field, reduxForm, propTypes } from 'redux-form'
+
+import {
+  input,
+  SubmitButton,
+} from 'components/form'
+import { required } from 'components/form/validators'
+import './organizationForm.css'
+
+const OrganizationForm = ({ type, ...formProps }) => (
```
Comment (bpetetot) [inline src/screens/organizer/organization/components/organizationForm/organizationForm.jsx]: the `type` is useless here, no ?

File: src/screens/organizer/organization/create/organizationCreate.container.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import forRoute from 'hoc-little-router'
+
+import OrganizationForm from '../components/organizationForm'
+
+const FORM_NAME = 'organization-create'
+const select = formValueSelector(FORM_NAME)
+
+const mapStore = store => ({
+  form: FORM_NAME,
+  type: select(store.getState(), 'type'),
```
Comment (bpetetot) [inline src/screens/organizer/organization/create/organizationCreate.container.js]: here the type is also useless...

**Review by bpetetot — CHANGES_REQUESTED** [review]

**Review by bpetetot — APPROVED** [review]
👌 Perfect ;)

**Comment by frinyvonnick** [comment]
I forgot to add organization to user's organizations list 😢

**Comment by bpetetot** [comment]
Yes good catch ! I missed it too :P

**Review by bpetetot — APPROVED** [review]

**Pull request merged**